### PR TITLE
Mti 12394

### PR DESCRIPTION
**Pull request by hernan43**
https://jira.wiley.com/browse/MTI-12394

Added an 8.2 version of the Dockerfile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-docker_repo = learninghouse/nginx-phpfpm
+docker_repo = wiley/nginx-phpfpm
```
Comment (dhiller1) [inline Makefile:1]: Please forgive my silly question, but is there any concern of changing the repo name from learninghouse to wiley having an impact outside of here?

**Review by dhiller1 — COMMENTED** [review]
Only thing I was unsure of was my question on the renaming of the docker_repo

File: Makefile (hunk 1 above)
Comment (hernan43) [reply Makefile:1]: There shouldn't be but that was entirely accidental. I had it named that temporarily to help me during the development of the Dockerfile so it wouldn't clash with the actual learninghouse ones I was testing. I will change it back. Good catch!

**Comment by hernan43** [comment]
Just a note that there was an info.php file I was using to debug the PHP load out and settings. It got checked in by mistake and is now removed.

**Review by dhiller1 — APPROVED** [review]
LGTM

**Pull request merged**